src/renderer/components/chat/ChatListHelpers.tsx

```diff
@@ -24,17 +24,20 @@ export function debounceWithInit<ARGS extends Array<any>>(
   }
 }
 
-export function useMessageResults(queryStr: string | undefined) {
+export function useMessageResults(
+  queryStr: string | undefined,
+  chatId: number | null = null
+) {
   const [ids, setIds] = useState<number[]>([])
 
   const debouncedSearchMessages = useMemo(
     () =>
       debounceWithInit((queryStr: string | undefined) => {
-        DeltaBackend.call('messageList.searchMessages', queryStr || '', 0).then(
-          setIds
-        )
+        BackendRemote.rpc
+          .searchMessages(selectedAccountId(), queryStr || '', chatId)
```

Comment:
I dislike a bit that its searchMessages(accountId, query, chatId) instead of searchMessages(accountId, chatId, query)